### ISSUE TITLE
Revision for GHC 9.10 and 9.12

### ISSUE THREAD

**Issue by LaurentRDC** [phadej/aeson-optics#20]
Hello,

This package can be built using GHC 9.10 and 9.12, if one allows a higher upper bound on `base`.

Would you be able to create a revision on Hackage to allow newer versions of `base`?

Thank you!

**Comment by LaurentRDC** [comment]
@phadej Is support for newer versions of base for aeson-optics blocked in some way?

**Comment by phadej** [comment]
No, but I learned recently that 9.12.1 is *very* broken, so we probably should wait for 9.12.2. And I'd rather update then. THis package slipped through the cracks when I was updating stuff for GHC-9.10, sorry for that.

**Comment by LaurentRDC** [comment]
No worries, thanks for the quick response

**Comment by LaurentRDC** [comment]
Popping in to mention that GHC 9.12.2 was released and is now installable via GHCup

**Comment by msooseth** [comment]
Hey hey! Is there an update to this? 9.12.2 is now available: https://www.haskell.org/ghc/blog/20250314-ghc-9.12.2-released.html :)

**Comment by juhp** [comment]
This prevents Stackage Nightly from updating to hasql-1.9.3.2 by adding [testcontainers](https://hackage.haskell.org/package/testcontainers)

**Comment by juhp** [comment]
I think it would make sense to do a Hackage revision

**Comment by ingun37** [comment]
any updates?

**Comment by juhp** [comment]
Ping
